assets/style.css

```diff
@@ -27,7 +32,7 @@ body {
 .block>* {
 	position: absolute;
 	background-size: 128px;
-	background-image: url('/assets/stone_slab_side.png');
```

Comment:
these URLs were correct before - if you're having issues locally, run a http server rather than opening html files directly

could u revert these assets-based changes

Reply:
Reverted them, was mainly using them to make the site loadable from directories besides `/`.